example/src/main/tut/data/Maybe.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: docs
+title:  "Maybe"
+---
+
+# Maybe
+
+`Maybe` can either signal the presence or absence of a value and is a safer alternative to `scala.Option`.
+`Maybe` does not provide unsafe method such as `Option.get` and `Option.head`, both of which may throw exceptions.
+
+**Typical imports**
+
+```tut:silent
+import scalaz.Prelude._
+```
+
+## Creation
+
+```tut
+just("Hello World")
+empty[Int]
+```
+
+## Use the Monad instance
+
+```tut
+val x: Maybe[Long] = just(50L)
+x.flatMap { y =>
+  just(2 * y)
+}
+```
```

Comment:
Could show off `map` and `pure`, too. Also, as it's an opaque type alias, the existence of `Just` and `Empty` extractors may be a useful thing to point out.

Reply:
#1547